src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.BOTTOM_SHOOTER_ID;
+import static frc.robot.Constants.TOP_SHOOTER_ID;
+
+public class Shooter {
+
+    private final WPI_TalonSRX topShooter = new WPI_TalonSRX(TOP_SHOOTER_ID);
+    private final WPI_TalonSRX bottomShooter = new WPI_TalonSRX(BOTTOM_SHOOTER_ID);
+
+    public Shooter(){
+        bottomShooter.follow(topShooter);
+    }
+
+    public void runShooter(double shooterTop, double shooterBottom){
```

Comment:
I would change the argument names to topShooterSpeed, we need it to be readable and intuitive for the freshman / new programmers